R/tests/testthat/test_write_audio.R

```diff
@@ -56,10 +56,14 @@ test_that("The function creates and removes audio files correctly", {
   expect_true(all(file.exists(file.path(tmp_dir, audio_files))))
 
   # Remove the files created during the test
-  file.remove(file.path(tmp_dir, audio_files))
+  if (file.exists(file.path(tmp_dir, audio_files))) {
+    file.remove(file.path(tmp_dir, audio_files))
+    }
```

Comment:
We can remove this code to remove the files created during testing, since the next lines of code to remove the whole temporary directory will also remove these files